cocos/asset/asset-manager/downloader.ts

```diff
@@ -94,6 +95,7 @@ const downloadCCON = (url: string, options: Record<string, any>, onComplete: ((e
 };
 
 const downloadCCONB = (url: string, options: Record<string, any>, onComplete: ((err: Error | null, data?: CCON | null) => void)): void => {
```

Comment:
"Why is there still a `downloadCCONB` process even though `cconb` is no longer being output?"